sdk/storage/azure-storage-queue/azure/storage/queue/aio/_queue_service_client_async.py

```diff
@@ -418,6 +422,7 @@ def get_queue_client(
                 :dedent: 8
                 :caption: Get the queue client.
         """
+        queue_name = ""
```

Comment:
I think a better thing to do is just remove the None check on line 427 and just always assign `queue_name = queue.name`. If it is `None`, so be it. It's better than uninitialized. Maybe check the `QueueClient` constructor to see if we check that the name is not `None` since we pass the name in on line 438.

Reply:
That makes sense, and this change will also make it consistent with `_queue_service_client.py` (sync version)!